downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/FetchTestReactomeOntologyFiles.java

```diff
@@ -0,0 +1,137 @@
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.StandardOpenOption;
+import java.sql.Blob;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.persistence.MySQLAdaptor;
+
+/**
+ * 
+ * @author jcook
+ *		Explanation of this module:
+		We are taking from value of the 'ontology' attribute from test_reactome_XX.Ontology. This returns a blob that can be divided into 3 sections: pprj, pont, and pins.
+		Unfortunately, the blob isn't cleanly divided into 3 sections, as between sections 1 and 2 there are some extra content not needed.
+		The content of the pprj file appears first. We take all content from the blob until the string 'pprj_file_content' appears. This signifies the end of that file.
+		Next, we iterate through the lines of the blob until a second dateTime string appears  (signifying the start of the pont file) and subsequently 
+		add all content until the string 'pont_file_content' appears, signifying the end of the pont file.
+		Finally, the rest of the blob pertains to the 'pins' file, so the remaining content is appended to the pins file.
+ */
+
+public class FetchTestReactomeOntologyFiles {
+	private static final Logger logger = LogManager.getLogger();
+	private static Connection connect = null;
+	private static Statement statement = null;
+	private PreparedStatement preparedStatement = null;
+	private static ResultSet resultSet = null;
+	
+	public static void execute(MySQLAdaptor dba, String username, String password, String host, String database, String releaseNumber) throws SQLException, ClassNotFoundException, UnsupportedEncodingException, FileNotFoundException, IOException {
+		logger.info("Running FetchTestReactomeOntologyFiles step");
+		Class.forName("com.mysql.jdbc.Driver");
+		connect = DriverManager.getConnection("jdbc:mysql://" + host + "/" + database + "?" + "user=" + username + "&password=" + password);
+		statement = connect.createStatement();
+		resultSet = statement.executeQuery("SELECT ontology FROM Ontology");
+		// The returned value is a single blob composed of binary and text. The three files produced by this step (pprj, pins, pont) are found within this blob.
+		// A handful of regexes and conditional statements are used to handle this data and output the 3 files. 
+		String pprjFilename = "reactome_data_model.pprj";
+		String pontFilename = "reactome_data_model.pont";
+		String pinsFilename = "reactome_data_model.pins";
+		PrintWriter pprjWriter = new PrintWriter(pprjFilename);
+		PrintWriter pontWriter = new PrintWriter(pontFilename);
+		PrintWriter pinsWriter = new PrintWriter(pinsFilename);
```

Comment:
The PrintWriter objects don't seem to be used.  Can they be removed?  I think (but I'm not totally sure) the files will be created the first time `Files.write()` is called for each file.  Could you test it without the PrintWriter objects and see if it works?

Reply:
Created a 'createOutputFile' method that uses File isntead of PrintWriter -- the PrintWriter stuff is an artifact from early in my Java learning